rpcserver.go

```diff
@@ -2823,6 +2824,61 @@ func handleGetBlockTemplate(s *rpcServer, cmd interface{}, closeChan <-chan stru
 	}
 }
 
+// handleGetCoinSupply implements the getcoinsupply command.
+func handleGetCoinSupply(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	supply := int64(0)
```

Comment:
You should probably just do
var supply int64
here.  That is more like the rest of the code and int64 defaults to 0 already.